tests/integration-tests/test-suites/fullFlowDataQuery.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { setupAuditSourceTestData } from '../../shared-test-code/utils/aws/setupAuditSourceTestData'
+import { sendWebhookRequest } from '../../shared-test-code/utils/zendesk/sendWebhookRequest'
+import { getWebhookRequestDataForTestCaseNumberAndDate } from '../utils/getWebhookRequestDataForTestCaseNumberAndDate'
+import { pollNotifyMockForDownloadUrl } from '../../shared-test-code/utils/queryResults/getDownloadUrlFromNotifyMock'
+import { downloadResultsFileAndParseData } from '../../shared-test-code/utils/queryResults/downloadAndParseResults'
+
+describe('Data flows from audit bucket to output', () => {
+  beforeAll(async () => {
+    await setupAuditSourceTestData(
+      'encryptionPlainTextData.txt.gz',
+      'firehose/2022/05/01/01'
+    )
+  })
+
+  it('Should be able to query data that arose from either an encrypted or plain text data store', async () => {
+    const webhookRequest = getWebhookRequestDataForTestCaseNumberAndDate(
+      3,
+      '2022-05-01'
+    )
+    sendWebhookRequest(webhookRequest)
+    const downloadUrl = await pollNotifyMockForDownloadUrl(
+      webhookRequest.zendeskId
+    )
+    expect(downloadUrl.startsWith('https')).toBe(true)
+    const csvRows = await downloadResultsFileAndParseData(downloadUrl)
+    console.log('Got data ', csvRows)
```

Comment:
Was this for debugging?

Reply:
It's in a test, I think it's probably useful to see it coming out at the end (and it'll only appear once so it's not clogging the logs up).